src/core/getPluginApi.ts

```diff
@@ -0,0 +1,24 @@
+import {
+    PluginBrowserWindow,
+} from '../types';
+
+declare const window: PluginBrowserWindow
+export const getPluginApi = (uuid: string) => {
```

Comment:
The `getPluginApi` method is the one that provides access to the setters of UI Extensible Areas and also to the hooks.

The return of this method must be typed, so the developer will get an autocomplete when using it.